api-client/src/main/java/com/xing/api/data/messages/Conversation.java

```diff
@@ -26,135 +29,43 @@
 /**
  * Java representation of a Conversation.
  */
-public class Conversation implements Serializable {
+@AutoValue
+public abstract class Conversation implements Serializable {
```

Comment:
Strike that... in the email looked like, you where asking about the nullability.